azure-pipelines.yml

```diff
@@ -268,6 +262,16 @@ jobs:
       contents: '**\CodeSignSummary-*.md'
   - publish: $(Build.ArtifactStagingDirectory)\SiteExtension
     artifact: SiteExtension
+  - pwsh: |
+      if ((test-path $(Build.ArtifactStagingDirectory)\ZippedPatchSiteExtension))
+      {
+        Write-Host "Patched site extension detected."
+        Write-Host "##vso[task.setvariable variable=isPatchVersion;isOutput=true]true"
```

Comment:
>"##vso[task.setvariable variable=isPatchVersion;isOutput=true]true" [](http://example.com/codeflow?start=19&length=67)

Can we set this in `build-extensions.ps1` in line `201`?

Reply:
`build-extensions.ps1` is meant to have no knowledge of where it's running -- (it's something you run locally as well). So I don't want to start mixing devops behavior in that file if we don't have to.